.github/workflows/static_build.yml

```diff
@@ -75,8 +75,6 @@ jobs:
             ./micromamba --version
             ./micromamba --help
             ./micromamba
-            apt update && apt install ca-certificates
```

Comment:
I guess a simple `-y` for both commands would've fixed it

Reply:
Hmm, I'll try it out again. Why is `apt` not smart enough to detect on its own that it's in a non-interactive environment?

Reply:
🤷